packages/radix-vue/package.json

```diff
@@ -56,15 +56,31 @@
     "test": "vitest",
     "pub:release": "pnpm publish --access public"
   },
+  "peerDependencies": {
+    "@internationalized/date": "^3.5.2",
+    "vue": ">= 3.2.0"
+  },
+  "peerDependenciesMeta": {
+    "@internationalized/date": {
+      "optional": true
+    },
+    "vue": {
+      "optional": false
+    }
```

Comment:
I think adding vue in peerDependencies is enough cause the optional property is false by default

Reply:
yep, you are right. I removed it